src/App.js

```diff
@@ -185,7 +185,7 @@ class App extends Component {
           </BodyContainer>
         )}
 
-        {currentPanel === "selectTrainer" && (
```

Comment:
This will impact levelbuilder mode parameters, yes? Should we hold off on this change or coordinate on the Code Studio side to get it in? I think it will mean updating level params already in the wild.

Reply:
I'm not sure that it will affect any of the params in the wild.  My understanding is that everything that currently exists in curriculum is now inventoried in https://github.com/code-dot-org/ml-playground/pull/138, so we could verify against that.

Reply:
>My understanding is that everything that currently exists in curriculum is now inventoried in #138, so we could verify against that.

Hi. Eavesdropping. Yes - this is true - PR 138 now accurately reflects levelbuilder parameters. So if each of those new dropdown options looks good, then we should feel confident that this won't break anything on levelbuilder. And if it does, I can blame myself since I'm the one who made that PR =P